openpype/tools/project_manager/project_manager/widgets.py

```diff
@@ -451,3 +489,32 @@ def wheelEvent(self, event):
             event.ignore()
         else:
             super(DoubleSpinBoxScrollFixed, self).wheelEvent(event)
+
+
+class NumScrollWidget(SpinBoxScrollFixed):
+    def __init__(self, minimum, maximum):
+        super(NumScrollWidget, self).__init__()
+        self.setMaximum(maximum)
+        self.setMinimum(minimum)
+        self.setButtonSymbols(QtWidgets.QSpinBox.NoButtons)
+        if platform.system() == "Darwin":
+            self.setSizePolicy(
```

Comment:
I think you don't need to set size policy when you call `setMinimumWidth` -> it will change the policy automatically. Have you tried just this (without minimum width)?
```
self.setSizePolicy(
    QtWidgets.QSizePolicy.MinimumExpanding,
    QtWidgets.QSizePolicy.Fixed
)
```

Reply:
Yes, I think I tried all these variants including `MinimumExpanding`. It seems this is some kind og macos override for the forms. see [here](https://stackoverflow.com/a/56176637/3346965):

> By default, the line edits are not growing to fill the available space on macOS because they are not supposed to. This is some sort of macOS UI guideline.

For the forms it is [suggested](https://github.com/spyder-ide/spyder/issues/8344#issuecomment-444308305) to explicitly set the [FieldGrowthPolicy](http://doc.qt.io/qt-5/qformlayout.html#fieldGrowthPolicy-prop) of the `QFormLayout` to `QFormLayout.ExpandingFieldsGrow`. By default, it is set to FieldsStayAtSizeHint for MacOS. I'm not sure if this is relevant here.

Reply:
UPD
Yep, that was exactly the case. I have changed `setFieldGrowthPolicy` for `inputs_layout` to `AllNonFixedFieldsGrow`, and now layout is correct.

[60da64e](https://github.com/ynput/OpenPype/pull/5075/commits/60da64ef424d1c772f7a5c1699b52fa706da4274)